root/static/scripts/guess-case/modes.js

```diff
@@ -282,3 +287,21 @@ exports.Sentence = assign({}, DefaultMode, {
     {url: {href: 'https://musicbrainz.org/doc/Style/Language', target: '_blank'}}
   ),
 });
+
+exports.Turkish = assign({}, DefaultMode, {
+  description: l(
+    'This mode capitalises almost all words of a sentence, the words ' +
+    'which are restricted to be capitalised are lowercased. Some words, often proper nouns, ' +
+    'may need to be manually fixed according to the {url|Turkish ' +
+    'language guidelines}.',
+    {url: {href: 'https://musicbrainz.org/doc/Style/Language/Turkish', target: '_blank'}}
+  ),
+
+  isSentenceCaps() {
+    return false;
+  },
+
+  isTurkishMode(){
```

Comment:
What about defining functions `toLowerCase(s)` and `toUpperCase(s)` instead of `ìsTurkishMode()`?